From 1af9f383a9448da530de9f681e8fccf12435aac8 Mon Sep 17 00:00:00 2001
From: Li Yang <leoyang.li@nxp.com>
Date: Tue, 27 Aug 2019 19:03:59 -0500
Subject: [PATCH] sdk_dpaa: net: Rename skb_frag_t size to bv_len

Update for upstream data structure change similar to

commit b8b576a16f79efbdde49348147f491b176537d88
Author: Matthew Wilcox (Oracle) <willy@infradead.org>
Date:   Mon Jul 22 20:08:30 2019 -0700

    net: Rename skb_frag_t size to bv_len

Signed-off-by: Li Yang <leoyang.li@nxp.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_sg.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_sg.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_sg.c
@@ -960,7 +960,7 @@ int __hot skb_to_sg_fd(struct dpa_priv_s
 		frag = &skb_shinfo(skb)->frags[i - 1];
 		qm_sg_entry_set_bpid(&sgt[i], 0xff);
 		qm_sg_entry_set_offset(&sgt[i], 0);
-		qm_sg_entry_set_len(&sgt[i], frag->size);
+		qm_sg_entry_set_len(&sgt[i], frag->bv_len);
 		qm_sg_entry_set_ext(&sgt[i], 0);
 
 		if (i == nr_frags)
@@ -969,7 +969,7 @@ int __hot skb_to_sg_fd(struct dpa_priv_s
 			qm_sg_entry_set_final(&sgt[i], 0);
 
 		DPA_BUG_ON(!skb_frag_page(frag));
-		addr = skb_frag_dma_map(dpa_bp->dev, frag, 0, frag->size,
+		addr = skb_frag_dma_map(dpa_bp->dev, frag, 0, frag->bv_len,
 					dma_dir);
 		if (unlikely(dma_mapping_error(dpa_bp->dev, addr))) {
 			dev_err(dpa_bp->dev, "DMA mapping failed");
